README.md

```diff
@@ -125,42 +121,27 @@ Images for styling purposes are located inside `img/`, photos inside `img/media/
 
 ### Adding a project
 
-There are four types of projects: core/featured projects, regular projects, incubator projects, and macros.
+There are three types of projects: organization projects, affiliate projects, and core/featured projects.
 
-To add a regular project, create a new markdown file in the `_projects` folder with the following front matter:
+To add an organization project, insert a new entry, alphabetically, in the `_data/projects.yml` file with the following keys:
 
 ```yml
-layout: post
-title: "Cats"
-category: "Functional Programming"
-description: "An experimental library intended to provide abstractions for functional programming in Scala, leveraging its unique features. Design goals are approachability, modularity, documentation and efficiency."
-permalink: "https://non.github.io/cats/"
-github: "https://github.com/non/cats"
+- title: "Cats"
+  description: "An experimental library intended to provide abstractions for functional programming in Scala, leveraging its unique features. Design goals are approachability, modularity, documentation and efficiency."
```

Comment:
```suggestion
  description: "An experimental library that provides abstractions for functional programming in Scala, leveraging its unique features. Design goals are approachability, modularity, and efficiency."
```
optional... I would think "documentation" is sort of a minimum implicit requirement for all projects here and sounds odd to call it a "design goal", though  I acknowledge Cats' docs are among the best--and that this diff in your PR is sorta incidental.

Reply:
This feedback is good, however, this isn't actually the place to change the description of `cats` that would be in the `_data/projects.yml` file, and I'd like to avoid yak shaving that here as well.

So I've just updated this README snippet to match the current content in the data file.